internal/connector/internal/config/processors.go

```diff
@@ -0,0 +1,27 @@
+package config
+
+import (
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/environments"
+	"github.com/spf13/pflag"
+)
+
+type ProcessorsConfig struct {
+	ProcessorsEnabled bool `json:"processors_enabled"`
+}
+
+var _ environments.ConfigModule = &ProcessorsConfig{}
+
+func NewProcessorsConfig() *ProcessorsConfig {
+	return &ProcessorsConfig{
+		ProcessorsEnabled: false,
+	}
+}
+
+func (c *ProcessorsConfig) AddFlags(fs *pflag.FlagSet) {
+	fs.BoolVar(&c.ProcessorsEnabled, "processors-enabled", c.ProcessorsEnabled, "Enable support for (Smart Event) Processors")
+}
+
+func (c *ProcessorsConfig) ReadFiles() error {
+	// There ae no external configuration files for Processors
```

Comment:
@manstis We'll need some form of configuration for Processors in terms of revision, container image and perhaps channels. We've spoken briefly in the chat about this, but I think exact structure will depend on whether we see value in supporting `channel` for Processors from the start. Happy to sync offline about the structure so that we can make a final decision 👍

Reply:
channels, images and co will come from a catalog at some point right ?

Reply:
@lburgazzoli Yep, that's the plan 👍 We're trying to be as closely aligned to the Connector catalog concept as possible, even though we only have a single Processor type right now.

Reply:
Changes made to import a "Processor Catalog" using the same approach as taken by Connectors.